src/model/types/index.ts

```diff
@@ -336,6 +336,13 @@ export type GetInfoResponse = {
 	motd?: string;
 	parameter: { peg_out_only: boolean };
 };
+/**
+ * Response from mint at /info endpoint
```

Comment:
This is from the /restore endpoint.
```suggestion
 * Response from mint at /restore endpoint
```